src/pages/Chat/ChatConversation.tsx

```diff
@@ -28,103 +30,199 @@ export const ChatConversation: React.SFC = () => (
       >
         <Message id={ChatStep.INITIAL}>
           {({ appear }) => (
-            <ChatMessage
-              appear={appear}
-              typingDuration={2500}
-              onTyped={() => goToStep(ChatStep.NAME_INPUT)}
+            <Mufflable
+              muffled={
+                ![
```

Comment:
Oh JavaScript, why don't you have a `$value in $array` syntax 😞

Reply:
i know ):

Reply:
and `array.includes` isn't even part of es5 🤘